src/darwin/Framework/CHIP/MTRDevice_Concrete.mm

```diff
@@ -4723,8 +4745,21 @@ - (BOOL)_deviceHasActiveSubscription
     return HaveSubscriptionEstablishedRightNow(_internalDeviceState);
 }
 
+// TODO: make this configurable - for now use 500ms or 0.5 seconds
+#define MTRDEVICE_ACTIVE_COMMUNICATION_THRESHOLD_SECONDS (0.5)
+
 - (void)_deviceMayBeReachable
 {
+    // Ignore this call if actively receiving communication from this device
+    {
+        std::lock_guard lock(self->_lock);
+        NSTimeInterval intervalSinceDeviceLastActive = -[self.lastSubscriptionActiveTime timeIntervalSinceNow];
```

Comment:
Should only be doing this if self.lastSubscriptionActiveTime is not nil, right? Because it it is, `intervalSinceDeviceLastActive` ends up 0 per this formula, I think, and we will incorrectly return early.

Reply:
Right thank you for catching this. I'll fix this

Reply:
Done.